app/models/user.rb

```diff
@@ -13,7 +13,7 @@ class User < ApplicationRecord
   # cause enum already takes care of this validation too
   enum role: { provider: 'provider', renter: 'renter' }
 
-  validates :first_name, :last_name, presence: true
+  validates :first_name, :last_name, :profile, presence: true
```

Comment:
FYI - I think if we force profile to be present it will break our seeds for users :) So maybe better to remove it for now? (or change the seeds with some profile sentences?)

Reply:
alright! :)